lib/src/generateCPU.cc

```diff
@@ -307,6 +307,35 @@ void genNeuronFunction(const NNmodel &model, //!< Model description
                         }
                     }
 
+                    if (n.second.isInjected()) {
```

Comment:
I think this block is exactly the same (aside from dd_ prefixes and stuff) so could you move it into ``StandardGeneratedSections`` to save duplication between the two CPU and CUDA code generators.

Sorry for how painful adding stuff to the code generators is - a refactor is on its way!